nevergrad/functions/gym/test_gym_multi.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+import pytest
+from unittest import SkipTest
+import os
+from . import gym_multi
+
+
+def test_gym_multi() -> None:
+    for env_name in gym_multi.GymMulti.ng_gym:
+        assert env_name in gym_multi.GymMulti.env_names, f"{env_name} unknown!"
+        assert env_name not in gym_multi.NO_LENGTH, f"{env_name} in no length and in ng_gym!"
+    for env_name in gym_multi.GUARANTEED_GYM_ENV_NAMES:
+        assert env_name in gym_multi.GymMulti.env_names, f"{env_name} should be guaranteed!"
+    assert len(gym_multi.GYM_ENV_NAMES) >= 26 or os.name == "nt"
+
+
+def test_compiler_gym() -> None:
+    if os.environ.get("CIRCLECI", False):
+        raise SkipTest("No HTTP request in CircleCI")
```

Comment:
as mentioned above, this should go in the function directly, then no need to copy it everywhere